packages/sdk-v2/src/sdk/base/react.ts

```diff
@@ -58,7 +61,7 @@ export const useGetContract = (
   type?: SdkTypes,
   requestedChainId?: number
 ) => {
-  const sdk = useSDK(readOnly, type, requestedChainId);
+  const sdk = useSDK(readOnly, "base", requestedChainId);
   return useMemo(() => sdk?.getContract(contractName), [contractName, , sdk]);
```

Comment:
i dont see the problem here. if there isnt a problem then it is micro optimizations

Reply:
i see problem because useMemo doesn't guarantee 100% memoization, sometimes it could re-run callback if react engine requires this.
so for the lazy logic e.g. if not created - create once useMemo isn't an otion